Block/Redirect/Redirect.php

```diff
@@ -102,6 +102,50 @@ public function __construct(
         $this->_taxCalculation = $taxCalculation;
     }
 
+	/**
+	 * Returns if the payment should follow the old HPP or the new Checkout flow
+	 *  - hpp will submit a form with all the additional information that the API requests\
+	 *  - checkout will redirect to a url without form submission
+	 *
+	 * @return bool
+	 * @throws \Exception
+	 */
+    public function isCheckoutAPM() {
```

Comment:
let's make sure that all alternative payment methods are routed throught the HPP. Separate task to route this to checkout.